ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -186,9 +190,9 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
                 className={classes.viewWorkersButton}
                 variant="outlined"
                 component={Link}
-                to={`${this.props.match.path}/${encodeURIComponent(
```

Comment:
`${this.props.match.path}` was used to not hardcode paths

Reply:
The new path goes all the way back to the root, so nothing of the matched route is preserved.  Is there a better way to generate the path for another view?

Reply:
So the `/provisioners/...` link leads to a different tool? The easiest thing to do is to hardcode then, in the absence of url constants

Reply:
It does.